taskcluster/ci/docker-image/kind.yml

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.docker_image:transforms
+    - taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+jobs:
+    python3.10:
+        definition: python
+        args:
+            PYTHON_VERSION: "3.10"
+    python3.9:
+        definition: python
+        args:
+            PYTHON_VERSION: "3.9.7"
```

Comment:
🤔 any reason why only 3.9 has minor version specified?

Reply:
minor nit: reorder desc or asc.

Reply:
Leftover from https://github.com/mozilla-releng/scriptworker-scripts/pull/428 :) I think we had a 3.9 watershed for winsign deps, osslsigncode maybe. Not applicable here, so I can remove.